File: src/main/java/data/input/BinFileReader.java
package data.input;

import data.Line;

import java.io.BufferedInputStream;
import java.io.FileInputStream;
import java.io.InputStream;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;

public class BinFileReader {
    public static Line ReadFromFile(String filename) {
        Path path = Paths.get(filename);
            //Files.readAllBytes(path);
        Line line = new Line(1);

        return  line;
        //InputStream inputStream = new BufferedInputStream(new FileInputStream(inputFile));
    }
}

File: src/main/java/data/input/BinFileReader.kt
package data.input

import data.Line

import java.nio.file.Path
import java.nio.file.Paths

object BinFileReader {
    fun readFromFile(filename: String): Line {
        val path = Paths.get(filename)
        //Files.readAllBytes(path);

        return Line(1)
        //InputStream inputStream = new BufferedInputStream(new FileInputStream(inputFile));
    }

    fun saveToFile(filename: String) {
        val path = Paths.get(filename)
        //Files.readAllBytes(path);
        //InputStream inputStream = new BufferedInputStream(new FileInputStream(inputFile));
    }
}
